README.md

```diff
@@ -89,6 +89,13 @@ Examples of `Build` resource using the example strategies shipped with this oper
   $ hack/install-tekton.sh
   ```
 
+* Set the following environment variables to match your destination image:
+```
+export MY_REGISTRY=quay.io
+export MY_USERNAME=adamkaplan
+export MY_REPO=shipwright-example-nodejs
+```
+
```

Comment:
Please move this instruction before the "Create a Cloud Native Buildpacks" step, and change the values so they are clearly fake:

```suggestion
export MY_REGISTRY=my-registry.io
export MY_USERNAME=my-username
export MY_REPO=my-image-repo
```

Reply:
I will create an issue so we can unify the different `ENV` that we use through `Makefile` and `shell scripts`  or `substenv` because we can have discrepancies as developers when using `make install` and not specifying correctly the target repositories to test.

Reply:
fixed otherwise